cockatrice/src/userconnection_information.cpp

```diff
@@ -0,0 +1,72 @@
+#include "userconnection_information.h"
```

Comment:
You need to add this file to CMakelists.txt in this list: https://github.com/Cockatrice/Cockatrice/blob/master/cockatrice/CMakeLists.txt#L117 for it to be picked up and compiled.